package.json

```diff
@@ -52,6 +52,7 @@
   },
   "peerDependencies": {
     "@fortawesome/fontawesome-free": "5.7.2",
+    "color": "^3.1.0",
```

Comment:
Need to be remove here: https://github.com/scality/core-ui/pull/8/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R38?